test/GameSecretTest.cs

```diff
@@ -101,43 +101,16 @@ public void TestToBytes()
 		[Test]
 		public void TestEquals()
 		{
-			var s2 = new GameSecret()
-			{
-				Region = GameRegion.US,
-				TargetGame = Game.Ages,
-				GameID = 14129,
-				Hero = "Link",
-				Child = "Pip",
-				Animal = Animal.Dimitri,
-				Behavior = 4,
-				IsLinkedGame = true,
-				IsHeroQuest = false,
-				WasGivenFreeRing = true
-			};
-
-			Assert.AreEqual(DesiredSecret, s2);
+			Assert.True(new GameSecret().Equals(new GameSecret()));
 		}
 
 		[Test]
 		public void TestNotEquals()
 		{
-			var s2 = new GameSecret()
-			{
-				Region = GameRegion.US,
-				TargetGame = Game.Seasons,
-				GameID = 14129,
-				Hero = "Link",
-				Child = "Pip",
-				Animal = Animal.Dimitri,
-				Behavior = 4,
-				IsLinkedGame = true,
-				IsHeroQuest = false,
-				WasGivenFreeRing = true
-			};
-
-			Assert.AreNotEqual(DesiredSecret, s2);
-			Assert.AreNotEqual(DesiredSecret, null);
-			Assert.AreNotEqual(DesiredSecret, "");
+			Assert.AreNotEqual(DesiredSecret, new GameSecret());
+			Assert.False(new GameSecret().Equals(new TestSecret()));
+			Assert.False(new GameSecret().Equals(new MemorySecret()));
+			Assert.False(new GameSecret().Equals(null));
```

Comment:
You have a mix of code styles here. For consistency with existing code, use the `Assert.AreNotEqual` method instead, unless you want to rewrite everything to use the newer `Assert.That` style documented here: https://github.com/nunit/docs/wiki/Assertions